configs/components/recom/recom.yaml

```diff
@@ -45,123 +45,119 @@ scenario: "preindustrial"
+            REcoM_PI: "${REcoM_PI}"
+        paco2_flux_param:
+            CO2_for_spinup: "${CO2_for_spinup}"
 
 
 choose_lresume:
-        true:
-                add_namelist_changes:
-                        namelist.recom:
-                                pavariables:
-                                        recom_restart: true
-        false:
-                add_namelist_changes:
-                        namelist.recom:
-                                pavariables:
-                                        recom_restart: false
+    true:
+        add_namelist_changes:
+            namelist.recom:
+                pavariables:
+                    recom_restart: true
+    false:
+        add_namelist_changes:
+            namelist.recom:
+                pavariables:
+                    recom_restart: false
 
 config_files:
-        recom:  recom
+    recom:  recom
 
 config_sources:
-        recom:  "${namelist_dir}/namelist.recom"
-#        io.recom.ciso: "${namelist_dir}/namelist.io.recom.ciso"
+    recom: "${namelist_dir}/namelist.recom"
+    # io.recom.ciso: "${namelist_dir}/namelist.io.recom.ciso"
 
 restart_name: ""
 
 restart_in_files:
-        bio_restart: bio_restart
+    bio_restart: bio_restart
 
 restart_in_in_work:
-        bio_restart: fesom.${parent_date!syear}.bio.${restart_name}nc
+    bio_restart: fesom.${parent_date!syear}.bio.${restart_name}nc
 
 restart_in_sources:
-        bio_restart: fesom.${parent_date!syear}.bio.nc
+    bio_restart: fesom.${parent_date!syear}.bio.nc
 
 restart_out_files:
-        bio_restart: bio_restart
+    bio_restart: bio_restart
 
 restart_out_in_work:
-        bio_restart: fesom.${end_date!syear}.bio.${restart_name}nc
+    bio_restart: fesom.${end_date!syear}.bio.${restart_name}nc
 
 restart_out_sources:
-        bio_restart: fesom.${end_date!syear}.bio.nc
-
-
-yearly_outputs: []
-
-monthly_outputs: [diags3d01,diags3d02,CO2y,aCO2,rDIN,rDON,rDOC,rDSi,PAR,NPPn,NPPd,GPPn,GPPd,NNAn,NNAd,GNAn,GNAd,benCalc,benSi,benC,benN,denb,aN,aFe,Hp,CO2f,dpCO2s,pCO2s,bgc01,bgc02,bgc03,bgc04,bgc05,bgc06,bgc07,bgc08,bgc09,bgc10,bgc11,bgc12,bgc13,bgc14,bgc15,bgc16,bgc17,bgc18,bgc19,bgc20,bgc21,bgc22,slp,age01,Kw,alphaCO2,O2f]
-
-daily_outputs: []
-
-12_step_outputs: []
-
-outdata_files:
-        "[[yearly_outputs-->OUTPUT]]": OUTPUT
-        "[[monthly_outputs-->OUTPUT]]": OUTPUT
-        "[[daily_outputs-->OUTPUT]]": OUTPUT
-outdata_in_work:
-        "[[yearly_outputs-->OUTPUT]]": OUTPUT_fesom_${start_date!syear!smonth!sday}.nc
-        "[[monthly_outputs-->OUTPUT]]": OUTPUT_fesom_${start_date!syear!smonth!sday}.nc
-        "[[daily_outputs-->OUTPUT]]": OUTPUT_fesom_${start_date!syear!smonth!sday}.nc
-outdata_sources:
-        "[[yearly_outputs-->OUTPUT]]": OUTPUT_fesom_${start_date!syear!smonth!sday}.nc
-        "[[monthly_outputs-->OUTPUT]]": OUTPUT_fesom_${start_date!syear!smonth!sday}.nc
-        "[[daily_outputs-->OUTPUT]]": OUTPUT_fesom_${start_date!syear!smonth!sday}.nc
```

Comment:
I tried this. To make it clean I think we need the merge-lists-functionality (#672) because in the end all (fesom and recom) variables need to get exported to the environment variable `AWI_FESOM_YAML`. The drawback is that all (fesom and recom) output will be moved to `outdata/fesom` but this is ok. Currently, all recom output is saved twice! (in outdata/fesom and outdata/recom).